torch/_subclasses/fake_tensor.py

```diff
@@ -143,6 +143,14 @@ def check_non_fake_tensor(x):
         r = run_function(func, types, (), new_kwargs)
         return FakeTensor(r, out_device)
 
+    if func in (aten.to.prim_Device, aten.to.device):
+        _, new_kwargs = normalize_function(func, args, kwargs, normalize_to_only_use_kwargs=True)
+        input_device = new_kwargs["device"]
+        out_device = input_device if input_device else new_kwargs["input"].device
+        new_kwargs["device"] = torch.device("meta")
+        r = run_function(func, types, (), new_kwargs)
+        return converter(r, out_device)
```

Comment:
For my understanding, how are we finding these special cases? Like have we split native functions in some way or is it from errors that dynamo has found by testing this?